helm-chart/renku-notebooks/values.yaml

```diff
@@ -21,6 +21,13 @@ gitlab:
 
 replicaCount: 1
 
+## Configure autoscaling
+autoscaling:
+  enabled: false
```

Comment:
I'm wondering if we should fully adopt the bitnami convention and have two values files in each chart. A `values.yaml` and a `production-values.yaml` which would have autoscaling on per default and a minimum of two replicas. Or maybe it's enough to do this on the RenkuLab chart level?

Reply:
yes, I think it's enough to do it there